superduperdb/db/mongodb/data_backend.py

```diff
@@ -51,6 +51,54 @@ def unset_outputs(self, info: t.Dict):
         select = Serializable.deserialize(info['select'])
         logging.info(f'unsetting output field _outputs.{info["key"]}.{info["model"]}')
         doc = {'$unset': {f'_outputs.{info["key"]}.{info["model"]}': 1}}
-
         update = select.update(doc)
         return self.db[select.collection].update_many(update.filter, update.update)
+
+    def delete_vector_index(self, vector_index):
+        collection = vector_index.indexing_listener.select.collection.name
+        self.db.command(
+            {
+                "dropSearchIndex": collection,
+                "name": vector_index.identifier,
+            }
+        )
+
+    # ruff: noqa: E501
+    def create_vector_index(self, vector_index):
+        collection = vector_index.indexing_listener.select.collection.name
+        key = vector_index.indexing_listener.key
+        model = vector_index.indexing_listener.model.identifier
+        self.db.command(
+            {
```

Comment:
Excuse my ignorance, but where does the schema for this data structure come from exactly? Useful to have some reference in a comment.